ts/features/bonus/bpd/screens/iban/IbanCTAEditScreen.tsx

```diff
@@ -38,6 +42,15 @@ const loadLocales = () => ({
  * Landing screen from the CTA message that asks to review user's IBAN insertion
  */
 const IbanCTAEditScreen: React.FC<Props> = (props: Props) => {
+  if (!props.bpdRemoteConfig?.program_active) {
+    Alert.alert(
+      I18n.t("bonus.bpd.title"),
+      I18n.t("bonus.bpd.iban.bpdCompletedMessage")
+    );
+    const navigation = useNavigationContext();
```

Comment:
From [Rules of Hooks](https://reactjs.org/docs/hooks-overview.html#rules-of-hooks):

> Don’t call Hooks inside loops, conditions, or nested functions.

I suggest you move it before the `if` block.

Reply:
Ops, you're right, changed in e491224